build/linux/installer/conf/fluent-bit-common.conf

```diff
@@ -77,10 +91,25 @@
     Match oms.container.log.flbplugin.mdsd.qos.*
     regex log ^MaODSRequest,https.*_LogManagement_LINUX_SYSLOGS_BLOB
 
+[FILTER]
+    Name grep
+    Alias ama-logs_amaca_log_tail
+    Match oms.container.log.flbplugin.amaca.*
+    Exclude log Information
+
 [OUTPUT]
     Name                            oms
     Alias                           oms_output
     EnableTelemetry                 true
     Retry_Limit                     10
     TelemetryPushIntervalSeconds    300
     Match                           oms.container.*
+    #${OMS_STORAGE_TOTAL_LIMIT_SIZE_MB}
+
+[OUTPUT]
+    Name                            oms
+    Alias                           oms_input_plugin_output
+    EnableTelemetry                 true
```

Comment:
this section looks to be from the old version of input plugin. can you remove it? strange rebase did not catch it

Reply:
> this section looks to be from the old version of input plugin. can you remove it? strange rebase did not catch it

Fixed. can you please double check if there are any other issues.